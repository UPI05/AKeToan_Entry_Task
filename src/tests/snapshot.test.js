import React from 'react';
import renderer from 'react-test-renderer';

it('react snapshot', () => {
  const tree = renderer.create(<div>Demo</div>).toJSON();
  expect(tree).toMatchSnapshot();
});
