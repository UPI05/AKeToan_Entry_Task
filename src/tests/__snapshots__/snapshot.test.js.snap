// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react snapshot 1`] = `
<div>
  Demo
</div>
`;
